cmd/cri-containerd/cri_containerd.go

```diff
@@ -31,8 +31,9 @@ import (
 func main() {
 	o := options.NewCRIContainerdOptions()
 	o.AddFlags(pflag.CommandLine)
-	options.InitFlags()
+	o.InitFlags(pflag.CommandLine)
 
+	glog.V(2).Infof("Run cri-containerd  %#v", o)
```

Comment:
nit: looks like you have two spaces there which might be accidental?

Reply:
@ijc   done